clang/test/3C/complexinlinestruct.c

```diff
@@ -6,7 +6,8 @@
 
  /* one decl; x rewrites to _Ptr<int> */
 struct foo { int *x; } *a;
-//CHECK: struct foo { _Ptr<int> x; } *a;
+//CHECK_NOALL: struct foo { _Ptr<int> x; } *a;
+//CHECK_ALL: _Ptr<struct foo> a = ((void *)0);
```

Comment:
I thought that in this case we should be able to rewrite `struct foo { int *x; } *a` as 
```
struct foo { int *x; }; 
_Ptr<struct foo> a;
```
Don't we do this sort of thing already? Sorry if I am misremembering.

Reply:
The point is: This is not an anonymous struct. It's a named struct. So it shouldn't be subject to the limitations on rewriitng.